planning/behavior_path_planner/docs/behavior_path_planner_avoidance_design.md

```diff
@@ -604,31 +604,52 @@ namespace: `avoidance.`
-| object_check_shiftable_ratio           | [m]   | double | Vehicles around the center line within this distance will be excluded from avoidance target.                                                                                                                                           | 0.6           |
-| object_check_min_road_shoulder_width   | [m]   | double | Vehicles around the center line within this distance will be excluded from avoidance target.                                                                                                                                           | 0.5           |
-| object_last_seen_threshold             | [s]   | double | For the compensation of the detection lost. The object is registered once it is observed as an avoidance target. When the detection loses, the timer will start and the object will be un-registered when the time exceeds this limit. | 2.0           |
-| left_hand_traffic                      | [-]   | bool   | Flag to select left or right hand traffic. `TRUE: LEFT-HAND TRAFFIC / FALSE: RIGHT-HAND TRAFFIC`                                                                                                                                       | true          |
+| Name                                      | Unit | Type   | Description                                                                                                                                                                                                                            | Default value |
+| :---------------------------------------- | :--- | :----- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :------------ |
+| threshold_distance_object_is_on_center    | [m]  | double | Vehicles around the center line within this distance will be excluded from avoidance target.                                                                                                                                           | 1.0           |
+| object_ignore_distance_traffic_light      | [m]  | double | If the distance between traffic light and vehicle is less than this parameter, this module will ignore it.                                                                                                                             | 30.0          |
+| object_ignore_distance_crosswalk_forward  | [m]  | double | If the front distance between crosswalk and vehicle is less than this parameter, this module will ignore it.                                                                                                                           | 30.0          |
```

Comment:
Distance ahead or distance in front sounds better?

Reply:
I'll update parameter name in the other PR.